packages/keyring-snap-bridge/src/SnapKeyring.ts

```diff
@@ -585,7 +589,7 @@ export class SnapKeyring extends EventEmitter {
    * @param snapId - The Snap ID to check allowed origins for.
    * @throws An error if the URL's origin is not in the Snap's allowed origins.
    */
-  #validateRedirectUrl(url: string, snapId: SnapId) {
+  async #validateRedirectUrl(url: string, snapId: SnapId): Promise<void> {
```

Comment:
Seems non-necessary, but this is a private method, so making it `async` is ok here.